templates/zerver/help/logging-in.md

```diff
@@ -54,39 +102,22 @@ from the **Zulip** menu in the top menu bar.
 
 {end_tabs}
 
-For subsequent logins, see [switching between organizations](/help/switching-between-organizations).
-
-## Troubleshooting
-
-### I don't know my Zulip URL
-
-Some ideas:
-
-* If you know your organization is hosted on
-  [zulip.com](https://zulip.com), go to [find my
-  account](https://zulip.com/accounts/find/) and enter the email
-  address that you signed up with.
-
-* Try guessing the URL. Zulip URLs often look like `<name>.zulipchat.com`,
- `zulip.<name>.com`, or `chat.<name>.com` (replace `<name>` with the name of your
-  organization).
-
-* Ask your organization administrators for your Zulip URL.
+## Switch between organizations
 
-### I signed up with Google/GitHub auth and never set a password
+{!switching-between-organizations.md!}
 
-If you signed up using passwordless authentication and want to start logging
-in via email/password, you can
-[reset your password](/help/change-your-password).
+## Set or reset your password
 
-### I forgot my password
+If you signed up using passwordless authentication and want to start logging in
```

Comment:
This might convey that the passwordless authentication will stop working if they choose to set a password. Maybe we should say something like:

```suggestion
If you signed up using passwordless authentication and want to have the option to log in
```